### ISSUE TITLE
Chain "value" (coin supply) seems innacurate

### ISSUE THREAD

**Issue by pinheadmz** [handshake-org/hsd#394]
The coin supply chart at https://hnscan.com/charts/supply has indicated a recent tick-down in coin supply, which I'm investigating.

I wrote a script that iterates through the entire chain, [returning a total in/out/diff for each block and maintains a running grand total.](https://gist.github.com/pinheadmz/3ab8ac3dcd2d1ae6b7fc5e613ce043bb)

Observations:

1. no blocks returned a negative "diff" (miners are accumulating all available fees and no coins are being destroyed) 

2. compared to `getInfo` the chain state reports a totally different value:

### Script output
```
height:  6590 in:  2462599819738 out:  2464599819738 diff:     2000000000 grand total: 269816163166950
```

### ChainState
```
    state: {
      tx: 170307,
      coin: 108139,
      value: 262612642694412,
      burned: 1060028925848
    }
```

So according to chainstate, `7,203,520.472538 HNS` is somehow missing.
This isn't related to https://github.com/handshake-org/hsd/issues/380 but perhaps a PR to clean up both issues will be possible once this issue is found.

**Comment by chjj** [comment]
This is a bug in the total supply calculation (this affects the UI only). We can fix this with a simple data migration. The supply is lower than it should be as a result of subtracting some outputs' values that it shouldn't be. I'll start working on a migration now and have it for the next release.

**Comment by boymanjor** [comment]
Fixed by #396.
